includes/admin/class-wcmypa-admin.php

```diff
@@ -975,6 +975,7 @@ private function getConfirmationData(DeliveryOptions $deliveryOptions): ?array
         $onlyRecipientTitle = WCMP_Checkout::getDeliveryOptionsTitle(WCMYPA_Settings::SETTING_ONLY_RECIPIENT_TITLE);
         $hasSignature       = $deliveryOptions->getShipmentOptions()->hasSignature();
         $hasOnlyRecipient   = $deliveryOptions->getShipmentOptions()->hasOnlyRecipient();
+        $confirmationData   = null;
```

Comment:
zodra je `WCMYPA()->setting_collection-> .....` (zie andere comment) in een variabel heb staan dan zal je `$confirmationData   = null;` niet meer nodig hebben. Je kan de variable dan gebruiken bij de volgende if `if (! $deliveryOptions->getCarrier()) {`
Voorbeeld `if (! $deliveryOptions->getCarrier() || ! $deliveryOptionsEnabled) {`

Reply:
done